terraform/main.tf

```diff
@@ -0,0 +1,42 @@
+
+provider "google" {
+  project     = var.project_id
+  region      = var.region
+}
+
+# Cloud Run
+resource "google_cloud_run_service" "service" {
```

Comment:
we need to add DNS record mapping in here

Reply:
@timothy-amplication I think as it is a staging environment I think that the temp URL is sufficient enough, as long as the GH action outputs the URL value.

What do you think?

Reply:
@mshidlov I think having a `staging` record will definitely make our lives easier when needing to deploy/test different versions - especially if the frequency of deploys are high - automate the pain away! Also the cost of managing 2 additional records are negligible. If anything, at least the backend (blog-server) has a record, because if the backend changes, the frontend will fail to reach out to the old record.